solutions/devsprint-leonardo-santos-1/FinanceAppTests/Modules/Home/Doubles/HomeViewDataDelegateSpy.swift

```diff
@@ -0,0 +1,16 @@
+@testable import FinanceApp
+
+final class HomeViewDataDelegateSpy: HomeViewModelDelegate {
+    
+    private(set) var didFetchHomeDataTriggered: Bool = false
+    private(set) var didFetchHomeDataCount: Int = 0
+    private(set) var didFetchHomeDataReturn: HomeData? = .fixture()
```

Comment:
Quando a propriedade é relacionada ao que vai ser retornado não precisamos de `private(set)`. Nesse caso, o dev precisa ter acesso para poder mockar o cenário que quiser! 
```suggestion
    var didFetchHomeDataReturn: HomeData? = .fixture()
```